src/main/java/com/direwolf20/buildinggadgets/common/tools/InventoryManipulation.java

```diff
@@ -310,9 +309,9 @@ public static int countInContainer(IItemHandler container, Item item, int meta)
     }
 
     private static List<Integer> findItem(Item item, int meta, InventoryPlayer inv) {
-        List<Integer> slots = new ArrayList<Integer>();
-        for (int i = 0; i < 36; ++i) {
-            ItemStack stack = inv.getStackInSlot(i);
+        List<Integer> slots = new ArrayList<>();
+        for (int i = 0; i < inv.mainInventory.size(); i++) {
```

Comment:
Also, are you sure this encludes the offhand (the hotbar is presumably in there...)?

Reply:
It doesn't unfortunately and the thing that does is private on the class, if it needs include them then I can change how this works